tests/regressions/lcos/safely_destroy_cv_1481.cpp

```diff
@@ -0,0 +1,44 @@
+//  Copyright 2015 (c) Hartmut Kaiser
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+// This test case demonstrates the issue described in #1481:
+// Sync primitives safe destruction
+
+#include <hpx/hpx.hpp>
+#include <hpx/hpx_main.hpp>
+#include <hpx/include/local_lcos.hpp>
+#include <hpx/util/lightweight_test.hpp>
```

Comment:
Uhh, right. It still relies on the behavior of the condition_variable, though. Let's change the name of the test, though.